Cargo.toml

```diff
@@ -31,7 +31,7 @@ os-release = "^0.1.0"
 hostname = "^0.3.1"
 openssl = "^0.10.55"
 compact_jwt = { version = "0.3.5", optional = true }
-kanidm-hsm-crypto = { version = "^0.1.6", optional = true }
+kanidm-hsm-crypto = { version = "^0.1.7", optional = true }
```

Comment:
This is pending the changes in https://github.com/kanidm/hsm-crypto/pull/33 and the release of a new version of hsm-crypto.